.circleci/config.yml

```diff
@@ -50,7 +50,7 @@ jobs:
 
   flakiness:
     docker:
-      - image: itamararjuan/golang_with_node_aws
+      - image: circleci/golang:1.12.9
```

Comment:
How does that work for you? 
Looking at our flakiness script, it attempts to install some stuff with Node.js
So I don't think this can work unless you re-add steps that install Node.js on the Go container.

Can you share a bit more information as to why you are making this change?

Reply:
I think it only attempts to use NPM when running nightly.
Will revert anyway.